app/views/content/ways-to-train.md

```diff
@@ -52,7 +52,7 @@ keywords:
   - Teacher training courses
 ---
 
-You need 'qualified teacher status' (QTS) to work in the majority of schools in England including state maintained primary, secondary and special schools.
+You need ['qualified teacher status'](/what-is-qts) (QTS) to work in the majority of schools in England including state maintained primary, secondary and special schools.
```

Comment:
There are conflicts with the Ways to train that need resolving before I can check this change. @peteryates can you help? Suspect this may be to do with https://github.com/DFE-Digital/get-into-teaching-app/pull/2160?